Modix.Bot/Modules/DocumentationModule.cs

```diff
@@ -24,36 +27,49 @@ public DocumentationModule(DocumentationService documentationService)
             [Summary("The term to search for in the documentation.")]
                 string term)
         {
+
+            Regex reg = new Regex("^[0-9A-Za-z.<>]+$");
```

Comment:
Nit: if we're just doing a character-by-character validation, I don't think the `+` at the end does anything meaningful.